messages/btc.proto

```diff
@@ -114,6 +116,7 @@ message BTCSignNextResponse {
   bytes signature = 4;
   // Previous tx's input/output index in case of PREV_INPUT or PREV_OUTPUT, for the input at `index`.
   uint32 prev_index = 5;
+  AntiKleptoSignerCommitment anti_klepto_signer_commitment = 6;
```

Comment:
Same for the `HostNonceCommitment`, and possibly for the `AntiKleptoSignatureRequest`: so that the size constraints in the .option file (or the equivalent in Rust) does not have to be repeated.